scilifelab/report/delivery_notes.py

```diff
@@ -206,10 +206,32 @@ def _set_project_sample_dict(project_sample_item, source):
     return project_sample_d
 
 
+def _exclude_sample_id(exclude_sample_ids, sample_name, barcode_seq):
+    """Check whether we should exclude a sample id.
+
+    :param exclude_sample_ids: dictionary of sample:barcode pairs
+    :param sample_name: project sample name
+    :param barcode_seq: the barcode sequence
+
+    :returns: True if exclude, False otherwise
+    """
+    if exclude_sample_ids and sample_name in exclude_sample_ids.keys():
+        if exclude_sample_ids[sample_name]:
```

Comment:
I did not write this function, I am just using it :wink: . And yes, it is fine if it returns `None`. As we call the function in a `if` statement below, returning `None` will just result in not entering the `if` block which is exactly what we need :+1: